scripts/common/preflights.sh

```diff
@@ -638,3 +639,13 @@ allow_remove_docker_new_install() {
           logWarn "conflicts, please uninstall Docker and re-run the install script."
      fi
 }
+
+allow_migrate_from_rook_to_openebs() {
```

Comment:
following existing patterns, this function belongs in the openebs_pre_init functions of each openebs add-on version. do you have a reason for putting it here instead?

Reply:
If we add to each addon version that does not mean that it will ONLY fail when we try to add the addon?
So, should we allow apply the previous addons and etc prior fail?

Reply:
not having to change every old addon version, maybe?

Reply:
But I found a problem we need to check if we have kubectl first as well I am adding it.